packages/react-server-website/provisioning/deploy.yml

```diff
@@ -2,16 +2,19 @@
   hosts: tag_Name_react_server_box
   user: ubuntu
   tasks:
+    - name: Install Slack API token
+      script: decrypt_credentials.py -k slack.api.token > slack-api-token
```

Comment:
It is checked in: https://github.com/redfin/react-server/blob/master/packages/react-server-website/provisioning/decrypt_credentials.py.